setup.cfg

```diff
@@ -9,8 +9,9 @@ xfail_strict=true
 select=C, E, F, W, B, B901, B902
 
 # W503: line break before binary operator
-# W504 line break after binary operator
-ignore = W503, W504
+# W504: line break after binary operator
+# E203: space before/after :
+ignore = W503, W504, E203
```

Comment:
I don't think you need to ignore `E203` because it only checks for whitespace before ':' (not after) and there is nothing that fails with it ignored.